crates/matrix-sdk-ui/src/timeline/inner/state.rs

```diff
@@ -483,14 +483,14 @@ pub(in crate::timeline) struct TimelineInnerStateTransaction<'a> {
 }
 
 impl TimelineInnerStateTransaction<'_> {
+    #[instrument(skip_all, fields(limited = timeline.limited))]
```

Comment:
Is it worth logging for any log under `handle_sync_timeline` whether it was limited or not? I would think not, and having a single log at the top of this function's body would be sufficient to me, but no strong opinions here.

Reply:
If it's limited, we call `clear()` which now logs an event at debug level (before: trace). I think that's good enough (i.e. we don't care to have a log event that says the update was *not* limited, right?).

Reply:
I was wondering if, in the case of multiple logs statements (as is the case in this function's body, where there's a loop), it was relevant to have the information about the `limited` state for every single log. I think it would be a bit distracting, so we could remove logging of the field at the span (via instrument) level here. But no strong opinions.